app/packages/looker/src/overlays/util.ts

```diff
@@ -124,3 +124,11 @@ export const convertId = (obj: Record<string, any>): Record<string, any> => {
     })
   );
 };
+
+export const getHashLabel = (label: RegularLabel): string => {
+  if (label["index"] || label["index"] === 0) {
+    return `${label.label}.${label.index}`;
```

Comment:
If the label value changes, can we treat that as a new object? Will that block certain use cases?

Reply:
Keeping the color to only the `index` is helpful because it represents the object over time (the color will always be the same when the video is playing).

I'm don't think it should be considered a new object. That is a change in behavior from the original implementation. It is also unlikely that the label value changes, but for people that find value in coloring by instance, I think only using the `index` is best

Reply:
I think we should use `(label, index)` as the hash for color-by-instance. Since `index` is an integer and we show `vehicle    1` in Looker, I'm afraid that users will think that they should use `(vehicle, 1), (vehicle, 2), (animal, 1), (animal, 2)` naming convention, where indexes are only unique within a given `label` class.

Reply:
I'd still like to transition `index` to an ObjectId at some point, in which case we can drop `label` from the hash, since users would understand that these are supposed to be globally unique.